docs/source/help.rst

```diff
@@ -10,12 +10,13 @@ Discussion
 
 Conversation happens in the following places:
 
-1.  **General Usage questions** are directed to `StackOverflow`_ with the #featuretools tag.
-2.  **Bug reports and feature requests** are managed on the `GitHub issue
+1.  **General usage questions** are directed to `StackOverflow`_ with the #featuretools tag.
+2.  **Feature requests** can be made on the `Feature Request Board <http://feedback.featurelabs.com>`__.
+3.  **Bug reports** are managed on the `GitHub issue
     tracker`_.
-3.  **Chat** occurs on at `Slack`_. Please do not post general usage questions and bug reports
+4.  **Chat** occurs on `Slack`_. Please do not post general usage questions and bug reports
     on Slack. Instead people should use Stack Overflow or GitHub.
```

Comment:
Could it be more clear what Slack is for if not those two things?

Reply:
updated it a bit. we also have some guidelines within the "Asking for help" section